Detectors/Vertexing/src/SVertexer.cxx

```diff
@@ -513,11 +513,16 @@ void SVertexer::buildT2V(const o2::globaltracking::RecoContainer& recoData) // a
 
       // get Nclusters in the ITS if available
       int8_t nITSclu = -1;
+      bool shortOBtrack = false;
       auto itsGID = recoData.getITSContributorGID(tvid);
       if (itsGID.getSource() == GIndex::ITS) {
         if (isITSloaded) {
           auto& itsTrack = recoData.getITSTrack(itsGID);
           nITSclu = itsTrack.getNumberOfClusters();
+
+          if (itsTrack.hasHitOnLayer(6) && itsTrack.hasHitOnLayer(5) && itsTrack.hasHitOnLayer(4) && itsTrack.hasHitOnLayer(3)) {
+            shortOBtrack = true;
+          }
         }
       } else if (itsGID.getSource() == GIndex::ITSAB) {
         if (isITSTPCloaded) {
```

Comment:
Following this, I also renamed the bool into `shortOBITSOnlyTrack` to avoid that someone misunderstands the logic of the check that is actually done (which indeed is a bit hidden since ITSTPC tracks with 4 hits are always allowed no matter what)